rtt/CMakeLists.txt

```diff
@@ -99,6 +99,9 @@ OPTION(PLUGINS_ENABLE_SCRIPTING "Enable loading Orocos program scripts." ON)
 ### RTT-Typekit
 OPTION(PLUGINS_ENABLE_TYPEKIT "Enable default RTT typekit plugin." ON)
 
+### Orocos Device Interfaces
+OPTION(ENABLE_OROCOS_DEVICE_INTERFACES "Include Orocos Device Interface classes" OFF)
```

Comment:
Please add a check in `tests/CMakeLists.txt` to compile the `dev_test` conditionally. Otherwise it fails like in https://travis-ci.org/orocos-toolchain/rtt/builds/389208490#L1408.

Shouldn't the default be `ON` for backwards-compatibility?

Reply:
> Please add a check in `tests/CMakeLists.txt` to compile the `dev_test` conditionally. Otherwise it fails like in https://travis-ci.org/orocos-toolchain/rtt/builds/389208490#L1408.

Done in https://github.com/orocos-toolchain/rtt/pull/264/commits/930fcfacf45d7d73c8ac2b329093b2c20923e449.

> Shouldn't the default be `ON` for backwards-compatibility?

Indeed, I think it is not used anymore (#285).